sdks/python/apache_beam/utils/urns.py

```diff
@@ -177,7 +178,7 @@ def to_runner_api(self, context):
 
   @classmethod
   def from_runner_api(cls, fn_proto, context):
-    # type: (beam_runner_api_pb2.FunctionSpec, PipelineContext) -> Any
+    # type: (Type[RunnerApiFnT], beam_runner_api_pb2.FunctionSpec, PipelineContext) -> RunnerApiFnT
```

Comment:
This change indicates that we should return an instance of the class that this is called from, though it provides no runtime guarantees of this.